### ISSUE TITLE
Add yarn to base images

### ISSUE THREAD

**Issue by esemeniuc** [catthehacker/docker_images#8]
Github comes with yarn as part of the default runner (https://github.com/actions/virtual-environments/blob/main/images/linux/Ubuntu2004-README.md)

Please add yarn as it is used by quite a few open source projects :)

**Comment by catthehacker** [comment]
I don't want to pollute `act-` and `runner-` tags with programming language specific tools so I'm not going to add it there, but I want to create bigger images including more tools and targeted images as well.
I'll try to finish that this weekend.

**Comment by catthehacker** [comment]
available as
- `catthehacker/ubuntu:js-latest-dev`
- `catthehacker/ubuntu:js-20.04-dev`
- `catthehacker/ubuntu:js-18.04-dev`
- `catthehacker/ubuntu:js-16.04-dev`

**Comment by esemeniuc** [comment]
Awesome, thank you @catthehacker  :)

**Comment by catthehacker** [comment]
I would like to keep the issue open for now, since it's not automated, as well as I didn't test the image with any workflow so let me know if something is wrong in there

**Comment by bl-ue** [comment]
Is this available yet?  If I delete `~/.actrc`, run `act`, specify `Medium`, and run, it will fail:
```
.../tldr-pwa/workflow/2: line 1: yarn: command not found
```
https://github.com/bl-ue/tldr-pwa/blob/main/.github/workflows/ci.yaml

**Comment by catthehacker** [comment]
@bl-ue you need to specify it, it's not available via "first time run" config, `act -P ubuntu-latest=catthehacker/ubuntu:js-latest`

**Comment by bl-ue** [comment]
Hmm...sorry if this is dumb, but it's not working:
```console
❯ act -P ubuntu-latest=catthehacker/ubuntu:js-latest
[CI/Build & Deploy Site] 🚀  Start image=catthehacker/ubuntu:js-latest
Error: Error response from daemon: manifest for catthehacker/ubuntu:js-latest not found: manifest unknown: manifest unknow
```

**Comment by catthehacker** [comment]
@bl-ue nah, I'm dumb, forgot to add pushing non-versioned tags, will be fixed in #16
